scaffold/ddev/drainpipe.Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Install Task
+RUN sh -c "$(curl -fsSL https://taskfile.dev/install.sh)" -- -b /usr/local/bin -d $(cat .taskfile)
```

Comment:
Should this be referencing vendor/lullabot/drainpipe instead of .taskfile from the root directory? I think this was what had me confused about scaffolding out this file.